utils/outputwriter/outputwriter.go

```diff
@@ -2,16 +2,18 @@ package outputwriter
 
 import (
 	"fmt"
+	"strings"
+
 	"github.com/jfrog/froggit-go/vcsutils"
 	"github.com/jfrog/jfrog-cli-core/v2/utils/coreutils"
 	"github.com/jfrog/jfrog-cli-core/v2/xray/formats"
 	xrayutils "github.com/jfrog/jfrog-cli-core/v2/xray/utils"
-	"strings"
 )
 
 const (
 	FrogbotTitlePrefix                               = "[🐸 Frogbot]"
 	CommentGeneratedByFrogbot                        = "[JFrog Frogbot](https://github.com/jfrog/frogbot#readme)"
+	ReviewCommentGeneratedByFrogbot                  = "[🐸 JFrog Frogbot](https://github.com/jfrog/frogbot#readme)"
```

Comment:
We can combine `ReviewCommentGeneratedByFrogbot` and `CommentGeneratedByFrogbot`